differs/emerge_diff.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2018 Google, Inc. All rights reserved.
```

Comment:
these should be 2020. i'll merge this and fix myself, so we don't have to go back and forth.